packages/actor-abstract-mediatype-utilities/README.md

```diff
@@ -0,0 +1,15 @@
+# Comunica Abstract Binding Hash Actor
```

Comment:
This readme seems inconsistent with the package name.

Reply:
Also, (why) is this package needed?

Reply:
Because I found that the problem of looking up source types was one that reoccurred in a few different places. This way the implementation of looking for the source and comparing its type to a desired type is only implemented once. @joachimvh told me that I maybe could implement this behavior in an abstract module.

Reply:
Indeed, abstraction is good. But I think it might be overkill in this case (when you remove support for `KEY_CONTEXT_SOURCES `), as the LOC will be very small then I think. But let's have a look at it next time you're at the office.